app/src/main/java/app/revanced/manager/ui/screen/settings/BackupRestoreSettingsScreen.kt

```diff
@@ -54,7 +55,7 @@ import org.koin.androidx.compose.koinViewModel
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun ImportExportSettingsScreen(
+fun BackupRestoreSettingsScreen(
     onBackClick: () -> Unit,
     vm: ImportExportViewModel = koinViewModel()
```

Comment:
Please change the name of the associated navigation graph object and view model.